litmus-portal/frontend/src/containers/layouts/Header/ProfileDropdownItems.tsx

```diff
@@ -147,43 +153,40 @@ function ProfileInfoDropdownItems(props: ProfileInfoDropdownItemProps) {
             >
               Edit Profile
             </Button>
+
+            <Button
+              disabled={loggedOut}
+              size="small"
+              onClick={logOut}
+              classes={{ root: classes.buttonSignout }}
```

Comment:
What is the purpose of adding the key `root` here?

Reply:
Removed.